### ISSUE TITLE
Lower results

### ISSUE THREAD

**Issue by wanghai111** [xinntao/BasicSR#28]
I used matlab to reduce the resolution of the image, but why the test results obtained with the pre-training model are two points lower than the results given.

**Comment by xinntao** [comment]
Hi, @wanghai111 
What model do you use? and test on which dataset?

**Comment by xinntao** [comment]
Hi @wanghai111 
RRDB_ESRGAN_x4.pth is for best visual quality, but the PSNR may be low.
If you want to get higher PSNR, you are supposed to use `RRDB_PSNR_x4.pth`

**Comment by xinntao** [comment]
Hi @wanghai111 
Do you use GAN loss and perceptual loss? 
The high PSNR is achieved when you use the MSE or L1 (i.e. pixel-wise loss) loss. 
If you train the models under the GAN framework, the PSNR is low and does not reflect the visual quality. In other word, the PSNR is not a good metric for visual qualiy.

**Comment by xinntao** [comment]
Actually, it is a problem of perceptual-oriented SR approaches. Usually, we compare the images visually to see if we have reproduced or not. For example, when reproducing the SRGAN, we compare our results of Set14 with those produced by SRGAN.

**Comment by Kanbo0409** [comment]
@xinntao How to train the PSNR-oriented model? just use mse or L1 loss?

**Comment by xinntao** [comment]
Hi @Kanbo0409 
Yes, use MSE or L1 loss. See `Train SR models` in [Usage](https://github.com/xinntao/BasicSR#usage) for more details.

**Comment by xinntao** [comment]
Hi @ztt0821,
I have noticed this problem. I think the differences come from two aspects:
1) the differences between the MATLAB bicubic downsampling and the python implementation one. There are still some small values differing. I do not remember what version was used for the paper (maybe the python implementation).
2) I found some of the images were copped with mod 8. The borders bing difference.

I will update the results with more stand testing procedures later.
